packages/augur-ui/src/modules/market/components/market-trade-history/market-trade-history.tsx

```diff
@@ -70,7 +71,7 @@ export default class MarketTradeHistory extends Component<
                         />
                       </li>
                       <li>
-                        {priceTime.price.toFixed(4)}
+                        {priceTime.price.toFixed(isScalar ? 4 : 2)}
```

Comment:
there will be scalar markets with 0 decimals like sports markets `how many runs ...` or `how many wins ...` prob should look at numTicks to determine number of decimal places to show.